api/cluster.go

```diff
@@ -30,6 +30,7 @@ import (
 	pkgCluster "github.com/banzaicloud/pipeline/pkg/cluster"
 	pkgCommon "github.com/banzaicloud/pipeline/pkg/common"
 	"github.com/banzaicloud/pipeline/pkg/k8sclient"
+	pkgUtil "github.com/banzaicloud/pipeline/pkg/k8sutil"
```

Comment:
These kind of aliases should be avoided if possible. I don't see a colliding import, so I don't see the need.